tests/api/controllers/lidar_point_cloud/lidar_point_cloud.c

```diff
@@ -71,8 +71,12 @@ int main(int argc, char **argv) {
     float x = point_cloud[i].x;
     float y = point_cloud[i].y;
     float z = point_cloud[i].z;
-    float d = sqrt(x * x + y * y + z * z);
-    ts_assert_double_in_delta(image_lidar[i], d, 0.05, "Point cloud and range image does not correspond.");
+    if (!isinf(x) && !isinf(y) && !isinf(z)) {
+      float d = sqrt(x * x + y * y + z * z);
```

Comment:
If you are modifying this part of the code, maybe all these float variables could be set constant
```suggestion
      const float d = sqrt(x * x + y * y + z * z);
```

Reply:
I noticed a lot of places where `const` is missing, but I wanted to keep the diff small. Although, maybe it is better use the opportunity and improve the code.

Reply:
Yes, keeping the diff quite small is good and I would not change all the files.
I just noticed this block because it was highlighted in the diff.
This is not strictly related with this PR, so it's fine for me even if you prefer to keep it like this.